src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-block;
```

Comment:
after you give the parent `display: flex`, you cannot set `display` for direct children. i mean, technically you can, but it won't do anything
JFYI